Gruntfile.js

```diff
@@ -78,9 +86,10 @@ grunt.initConfig({
 });
 
 grunt.task.loadTasks('grunt-tasks/');
+grunt.loadNpmTasks('grunt-mocha');
 
 grunt.registerTask('dist', ['peg', 'eslint', 'babel:dist']);
-grunt.registerTask('acceptance', ['dist', 'babel:acceptance', 'browserify:test']);
+grunt.registerTask('acceptance', ['dist', 'babel:acceptance', 'browserify:test', 'mocha']);
```

Comment:
We could change this from `acceptance` to `test` now that it actually runs the tests.